contrib/executor/jmeterd/README.md

```diff
@@ -129,8 +131,50 @@ Testkube should now use the new image for the Executor and you can schedule a Te
+2. **SLAVES_COUNT**: Specifies the number of slave pods required for distributed JMeter tests. Example: `SLAVES_COUNT=3`. Default value of `SLAVE_COUNT` is 1.
 
 3. **MASTER_ADDITIONAL_JVM_ARGS / SLAVES_ADDITIONAL_JMETER_ARGS**: Allows exporting additional JVM arguments for slaves/master. Example: `MASTER_ADDITIONAL_JVM_ARGS=-Xmx1024m -Xms512m -XX:MaxMetaspaceSize=256m`.
 
 4. **SLAVES_ADDITIONAL_JMETER_ARGS**: Provides additional JVM arguments for JMeter server / slaves. Example: `SLAVES_ADDITIONAL_JMETER_ARGS=jmeter-server -Jserver.rmi.ssl.disable=true -Dserver_port=60000`.
+
+
+**Read README file of Jmeter to learn about how to run Jmeter test.**
+
+Below guide will provide you the details about how to run a Jmeter test in distribution environment.
+
+1. File option:
+
+   When you provide a test (.jmx) file to `Jmeterd ( Jmeter in distributed environment )`, the executor of `Jmeterd` will spawn number of slaves pods specified by user through `SLAVE_COUNT` Environment Variable as desribed above and run the test on all the slave pods.
+
+2. Git Option:
+   Using Git flow of the executor we can have following benifits of `Jmeterd` which is not possible:
+
+   - User can use additional file required by a particular test like a csv or json file.
+      There is an example of using a csv file by test (.jmx) file in the `example` folder of `Jmeterd`.
+   - User can provide dynamic plugins required for a Jmeter test by keeping the plugins inside the test folder in a directory named as `plugins`.
+   - User can also provide custom values of the paramters present in `user.properties` by using custom `user.properties` file in the git repo.
+
+
+   For using the Git Option and avail all the above features, user should have the following directory structure in the git repo:
+
+
+      github.com/`<username>/<reponame>`/---
+
+                                          |-test1/---
+                                                   |- testfile1.jmx
+                                                   |- userdata.csv ( or any other additional file )
+                                                   |- user.properties
+                                                   |- plugins/---
+                                                               |- plugin-manager.jar
+                                                               |- <jar file of any other required plugins to run test1>
+
+                                          |-test2/---
+                                                   |- testfile2.jmx
+                                                   |- userdata.json ( or any other additional file )
+                                                   |- user.properties
+                                                   |- plugins/---
+                                                               |- plugin-manager.jar
+                                                               |- <jar file of any other required plugins to run test2>
+
+
+
```

Comment:
Do we have this complete structure somewhere in this repo? If yes, we can just point that here instead of showing the directory tree here

Reply:
I am not sure whether we can show the Git repo structures starting from the `github.com` in the our repo. So that's why I have to show this dir structure starting from `github.com` so that it should be easy for user to understand.